manifests/install.pp

```diff
@@ -16,7 +16,6 @@
       password         => $crowd::password,
       password_min_age => '0',
       password_max_age => '99999',
-      managehome       => false,
```

Comment:
Unrelated change. If this needs to be optional, a separate PR or issue can be opened.